push

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+shopt nullglob
+
+OS=${OS:-fedora}
+VERSION=${VERSION:-40}
+OSVER=$OS-$VERSION
+WORKDIR=$PWD/work
```

Comment:
I was hoping for this but this will do as well...

```
TMPCFG=$(mktemp /tmp/nboci-files-XXXX)
trap "rm -rf $TMPCFG" EXIT
```

Reply:
I debated that (though I'd use `mktemp -t nboci-files-XXXX` to respect `$TMPDIR`), but in the end focused on other items. It already escalated quite a bit from what I originally wanted to fix.